### PR TITLE
libteec/Makefile: generate static library (libteec.a)

### PR DESCRIPTION
**Pull request by jforissier**
Signed-off-by: Jerome Forissier <jerome.forissier@linaro.org>

### CODE REVIEW

**Comment by jbech-linaro** [comment]
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

File: libteec/Makefile [hunk 1]
```diff
@@ -48,6 +48,10 @@ $(TEEC_LIBRARY): $(TEEC_OBJS)
 	$(VPREFIX)$(CC) -shared -Wl,-soname,$(LIB_MAJ_MIN) $(TEEC_LFLAGS) -o $@ $+
 	@echo ""
 
+$(OUT_DIR)/libteec.a: $(TEEC_OBJS)
+	@echo "  AR      $@"
+	$(VPREFIX)$(AR) rcs $@ $+
+
```
Comment (etienne-lms) [inline libteec/Makefile]: update the `clean` target.

Comment (jforissier) [reply libteec/Makefile]: Fixed, thanks

**Comment by etienne-lms** [comment]
`Reviewed-by: Etienne Carriere <etienne.carriere@linaro.org>`

**Pull request merged**